src/core/textures/Texture.js

```diff
@@ -216,15 +224,17 @@ export default class Texture extends EventEmitter
     {
         if (this.baseTexture)
         {
-            if (destroyBase)
+            // remove BaseTexture stored under the texture name if destroyBase is true or
+            // is our base texture (in which case this is the event callback for 'destroy')
+            if ((destroyBase === true || destroyBase === this.baseTexture)
```

Comment:
Not sure I understand what this is for. The docs weren't updated so I can only assume this is checking if you pass in a base texture? Why do we need to do this delete here and not just call baseTexture.destroy()? What do we buy from passing in a base texture as a param here, since it only does anything if the owned one is passed?